embulk-core/src/main/java/org/embulk/spi/AbstractReporterImpl.java

```diff
@@ -0,0 +1,23 @@
+package org.embulk.spi;
+
+import com.google.common.collect.ImmutableMap;
+import org.embulk.spi.Reporter;
+
+import java.util.Map;
+
+public abstract class AbstractReporterImpl
+        implements AutoCloseable, Reporter
+{
+    public final void reportLine(Level level, String line)
```

Comment:
Thank you. I will fix it. And we also change the method name `reportLine` with `reportString` as talked to you directly.